src/pages/Swap.tsx

```diff
@@ -97,7 +101,7 @@ function Swap(): ReactElement {
   const tokenBalances = usePoolTokenBalances()
   const basicPools = useContext(BasicPoolsContext)
   const tokens = useContext(TokensContext)
-  const { tokenSymbolToTokenMap, tokenSymbolToPoolNameMap } = useTokenMaps()
+  const { tokenSymbolToPoolNameMap } = useTokenMaps()
```

Comment:
will need to update to `tokenAddressToPoolNameMap`

Reply:
Done